src/_sass/_utilities.scss

```diff
@@ -0,0 +1,10 @@
+// taken from the a11y project example https://www.a11yproject.com/posts/how-to-hide-content/
+.visually-hidden {
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
```

Comment:
There's a [very similar class](https://github.com/dxw/playbook/blob/main/src/_sass/_accessibility.scss) in the playbook, which is also the one used on the dxw website. It has three extra declarations, so it might be worth adding those in here?

Reply:
ah excellent thank you, have replaced